db/postgreSQL/postgres_generate_csv.js

```diff
@@ -0,0 +1,91 @@
+const faker = require('faker');
+
+const fs = require('fs');
+
+const writeReviews = fs.createWriteStream('someBirdsReviews2.csv');
+writeReviews.write('name,headline,review,rating,fit_feedback,shoe_id\n');
+
+const writeShoes = fs.createWriteStream('someBirdsShoes2.csv');
+writeShoes.write('name,model,rating_average,fit_beedback_average,review_count\n');
+
+let randomNumberGenerator = (min, max) => {
```

Comment:
Put this util function in a separate module and require that module in this file.